bitnami/postgresql/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: postgresql
 sources:
   - https://github.com/bitnami/bitnami-docker-postgresql
   - https://www.postgresql.org/
-version: 11.2.6
+version: 12.2.6
```

Comment:
Why this version? At the moment it has a breaking change so it should be 12.0.0 not 12.2.6. Please have a look to [semver](https://semver.org/)

Reply:
You are totally right.